src/views/Header/Header.js

```diff
@@ -211,8 +211,18 @@ class Header extends React.PureComponent {
   };
 
   _renderRightComponent = props => {
-    const { headerRight } = props.scene.descriptor.options;
-    return headerRight || null;
+    const { options } = props.scene.descriptor;
+    const headerRight = options.headerRight;
+    if (React.isValidElement(headerRight) || headerRight === null) {
+      return headerRight;
+    }
+    if (!headerRight) return null;
+    return (
+      <headerRight
+        pressColorAndroid={options.headerPressColorAndroid}
```

Comment:
Is `options.headerPressColorAndroid` always defined?

Reply:
I don't know. But there is a `headerPressColorAndroid` in the `_renderLeftComponent()` function, which is used to define the press ripple color on Android. So I copied it to the `_renderRightComponent()` function. 
They both receive props from `props.scene.descriptor`.